src/Components/Form/FormFields/TextFormField.tsx

```diff
@@ -100,7 +100,7 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
       );
     const _trailing =
       trailing === trailingFocused ? (
-        <div className="absolute inset-y-0 right-0 flex items-center pr-3">
+        <div className="relative inset-y-0 right-0 flex items-center pr-3">
```

Comment:
Why was this changed? Unrelated to the issue being solved right?

Feel free to create a separate issue if this solves an issue 😄